src/LightGraphs.jl

```diff
@@ -3,7 +3,7 @@ module LightGraphs
 
 using Requires
 using GZip
-using StatsBase: binomrand
+using Distributions
```

Comment:
Yes, that's good practice and we should be encouraging explicit imports going forward.